letsencrypt/plugins/manual.py

```diff
@@ -151,6 +160,10 @@ def _perform_single(self, achall):
             if self._httpd.poll() is not None:
                 raise errors.Error("Couldn't execute manual command")
         else:
+            if not zope.component.getUtility(interfaces.IDisplay).yesno(
+                    self.IP_DISCLAIMER, "Yes", "No"):
+                raise errors.Error("Must agree to IP logging to proceed")
```

Comment:
I'd change this to `errors.PluginError` and write a test for it to fix [coverage](https://travis-ci.org/letsencrypt/letsencrypt/jobs/87273420#L838).